src/track/serato/markers.h

```diff
@@ -35,6 +35,10 @@ class SeratoMarkersEntry {
             bool hasEndPosition,
             int endPosition,
             SeratoStoredHotcueColor color,
+            int stem1vol,
```

Comment:
They are unused here and can be removed. 
Can you propose this change and we con look at the missing warning?

Reply:
I'll do,